### PR TITLE
add translations

### PR DESCRIPTION
**Pull request by dvm1984**
#### What is the purpose of this pull request?

<added translations

### CODE REVIEW

File: messages/de.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-  "shippingEstimate-bd": "{timeAmount, plural, =0 {Am selben Tag} one {In # Werktag} other {In bis zu # Werktagen}}",
-  "shippingEstimate-d": "{timeAmount, plural, =0 {Am selben Tag} one {In # Tag} other {In bis zu # Tagen}}",
-  "shippingEstimate-h": "{timeAmount, plural, =0 {Jetzt} one {In # Stunde} other {In bis zu # Stunden}}",
-  "shippingEstimate-m": "{timeAmount, plural, =0 {Jetzt} one {In # Minute} other {In bis zu # Minuten}}",
+  "shippingEstimate-bd": "{timeAmount, plural, =0 {Heute} one {In # Werktag} other {Bis zu # Werktagen}}",
+  "shippingEstimate-d": "{timeAmount, plural, =0 {Heute} one {In # Tag} other {Bis zu # Tage}}",
+  "shippingEstimate-h": "{timeAmount, plural, =0 {Jetzt} one {In# Stunden} other {Bis zu # Stunden}}",
+  "shippingEstimate-m": "{timeAmount, plural, =0 {Jetzt} one {In# Minute} other {Bis zu # Minuten}}",
```
Comment (lucasecdb) [inline messages/de.json]: should these spaces have been removed?

File: messages/de.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   "shippingEstimate-scheduled": "{date}, zwischen {startDate} und {endDate}",
   "shippingEstimate-scheduled-no-dates": "Geplant",
-  "shippingEstimatePickup-bd": "{timeAmount, plural, =0 {Versandfertig am selben Tag} one {Bereit in bis zu # Werktag} other {Bereit in bis zu # Werktagen}}",
-  "shippingEstimatePickup-d": "{timeAmount, plural, =0 {Versandfertig am selben Tag} one {Bereit in bis zu # Tag} other {Bereit in bis zu # Tagen}}",
-  "shippingEstimatePickup-h": "{timeAmount, plural, =0 {Bereit zur Abholung} one {Fertig in bis zu # Stunde} other {Fertig in bis zu # Stunden}}",
-  "shippingEstimatePickup-m": "{timeAmount, plural, =0 {Bereit zur Abholung} one {Fertig in bis zu # Minute} other {Fertig in bis zu # Minuten}}"
+  "shippingEstimatePickup-bd": "{timeAmount, plural, =0 {Heute fertig} one {Fertig in# Werktag} other {Fertig in# Werktagen}}",
+  "shippingEstimatePickup-d": "{timeAmount, plural, =0 {Heute fertig} one {Fertig in # Tag } other {Fertig in # Tagen}}",
```
Comment (lucasecdb) [inline messages/de.json]: extra space in the end of `one`

File: messages/sv.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "shippingEstimate-bd": "{timeAmount, plural, =0 {Idag} one {I # arbetsdag} other {Upp till # arbetsdagar}}",
+  "shippingEstimate-d": "{timeAmount, plural, =0 {Idag} one {I # dagar} other {Upp till # dagar}}",
+  "shippingEstimate-h": "{timeAmount, plural, =0 {Nu} one {I # timme} other {Upp till # timmar}}",
+  "shippingEstimate-m": "{timeAmount, plural, =0 {Nu} one {I # minut} other {Upp till # minuter}}",
+  "shippingEstimate-scheduled": "{date}, mellan {startDate} och {endDate}",
+  "shippingEstimate-scheduled-no-dates": "Schemalagd",
+  "shippingEstimatePickup-bd": "{timeAmount, plural, =0 {Redo idag} one {Redo i upp till # arbetsdag} other {Redo i upp till # arbetsdagar}}",
+  "shippingEstimatePickup-d": "{timeAmount, plural, =0 {Redo idag} one {Redo i upp till # dag} other {Redo i upp till # dagar}}\n​",
+  "shippingEstimatePickup-h": "{timeAmount, plural, =0 {Redo för upphämtning} one {Redo i upp till # timme} other {Redo i upp till # timmar}}\n​",
+  "shippingEstimatePickup-m": "{timeAmount, plural, =0 {Redo för upphämtning} one {Redo i upp till # minut} other {Redo i upp till # minuter}}\n​"
```
Comment (lucasecdb) [inline messages/sv.json]: newlines here

File: messages/de.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+  "shippingEstimatePickup-h": "{timeAmount, plural, =0 {Abholbereit} one {Abholbereit in# Stunde} other {Abholbereit in # Stunden}}",
+  "shippingEstimatePickup-m": "{timeAmount, plural, =0 {Abholbereit} one {Fertig in# Minute} other {Fertig in# Minuten }}"
```
Comment (lucasecdb) [inline messages/de.json]: also no space between the word and `#` character here

File: messages/de.json (hunk 1 above)
Comment (darlenedms) [reply messages/de.json]: Add them back, please.

File: messages/de.json [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
+  "shippingEstimate-d": "{timeAmount, plural, =0 {Heute} one {In # Tag} other {Bis zu # Tage}}",
+  "shippingEstimate-h": "{timeAmount, plural, =0 {Jetzt} one {In# Stunden} other {Bis zu # Stunden}}",
+  "shippingEstimate-m": "{timeAmount, plural, =0 {Jetzt} one {In# Minute} other {Bis zu # Minuten}}",
   "shippingEstimate-scheduled": "{date}, zwischen {startDate} und {endDate}",
   "shippingEstimate-scheduled-no-dates": "Geplant",
-  "shippingEstimatePickup-bd": "{timeAmount, plural, =0 {Versandfertig am selben Tag} one {Bereit in bis zu # Werktag} other {Bereit in bis zu # Werktagen}}",
-  "shippingEstimatePickup-d": "{timeAmount, plural, =0 {Versandfertig am selben Tag} one {Bereit in bis zu # Tag} other {Bereit in bis zu # Tagen}}",
-  "shippingEstimatePickup-h": "{timeAmount, plural, =0 {Bereit zur Abholung} one {Fertig in bis zu # Stunde} other {Fertig in bis zu # Stunden}}",
-  "shippingEstimatePickup-m": "{timeAmount, plural, =0 {Bereit zur Abholung} one {Fertig in bis zu # Minute} other {Fertig in bis zu # Minuten}}"
+  "shippingEstimatePickup-bd": "{timeAmount, plural, =0 {Heute fertig} one {Fertig in# Werktag} other {Fertig in# Werktagen}}",
```
Comment (darlenedms) [inline messages/de.json]: Add the spaces in this 2 too.

File: messages/de.json [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
+  "shippingEstimatePickup-d": "{timeAmount, plural, =0 {Heute fertig} one {Fertig in # Tag} other {Fertig in # Tagen}}",
+  "shippingEstimatePickup-h": "{timeAmount, plural, =0 {Abholbereit} one {Abholbereit in# Stunde} other {Abholbereit in # Stunden}}",
```
Comment (darlenedms) [inline messages/de.json]: Space.

File: messages/de.json [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
+  "shippingEstimatePickup-m": "{timeAmount, plural, =0 {Abholbereit} one {Fertig in# Minute} other {Fertig in# Minuten}}"
```
Comment (darlenedms) [inline messages/de.json]: Space.

File: messages/de.json (hunk 1 above)
Comment (lucasecdb) [reply messages/de.json]: ok!

**Review by darlenedms — APPROVED** [review]

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**